omero/sysadmins/cli/LightAdmins.txt

```diff
@@ -0,0 +1,61 @@
+Adjusting administrator restrictions
+====================================
+
+OMERO 5.4 introduced the concept of a :doc:`restricted administrator
+<../admins-with-restricted-privileges>`. The meaning and representation
+of the server's underlying permissions restrictions is described in
+:doc:`developer documentation</developers/Server/LightAdmins>`.
+OMERO.web offers easy management of restrictions and is recommended for
+setting up restricted administrators via its :help:`Admin tab
+<facility-manager#lightadmin>`.
+
+OMERO.cli does not offer easy management of restrictions because support
+is yet to be added. In the meantime it can already manipulate
+administrator restrictions in the awkward manner described hereunder.
+
+.. warning::
+
+  OMERO.web provides a simplified view of the available restrictions:
+  checking *one* box in the web interface may lift *multiple* underlying
+  restrictions from an administrator. The recommended OMERO.web
+  management interface may thus prove confusing if OMERO.cli has been
+  used to set a combination of restrictions that does not correspond to
+  those bundles of related restrictions available in OMERO.web.
+
+
+View an administrator's restrictions
+------------------------------------
+
+For an administrator with user ID 123, :omerocmd:`hql "SELECT ap.name
```

Comment:
I think the commands here should not be in-line. 
See for example ``sysadmins/cli/usergroup.html``:
![screen shot 2017-07-11 at 10 58 42](https://user-images.githubusercontent.com/2478303/28063154-1ffe980e-6628-11e7-9c6a-b852449218aa.png)
Even shorter one line commands profit from beiing inside a formatting box.